Jenkinsfile

```diff
@@ -31,11 +31,15 @@ void setBuildStatus(String context, String status) {
             break
     }
 
+    try {
     githubNotify credentialsId: 'creds-github-suse-cf-ci-bot',
                  context: "jenkins/${context}",
                  description: description,
                  status: status.toUpperCase(),
                  targetUrl: env.BUILD_URL
+    } catch (IllegalArgumentException e) {
+      echo "Can't notify github status (can't infer git data)"
```

Comment:
What is causing this? I thought it had something to do with credentials but in that case we would rather fix it than ignore it.

Reply:
Credentials; we're doing this for now to get builds working again, expecting a follow-up PR to do this correctly.